pyuvdata/uvdata/uvdata.py

```diff
@@ -2225,7 +2225,8 @@ def __setattr__(self, __name, __value) -> None:
             ):
                 warnings.warn(
                     f"phase_center_catalog is already set, so {__name}, which is an "
-                    "old phase attribute, cannot be set.",
+                    "old phase attribute, cannot be set. This will become an error in "
```

Comment:
Maybe worth adding to milestone list if not already.

Reply:
It's there already.